StreamingUpdatesRestApi/Program.cs

```diff
@@ -367,6 +380,45 @@ public static async Task<bool> MainAsync(bool test = false)
 
                     Console.WriteLine();
                     #endregion
+
+                    // Step 13
+                    // Create additional signups and make an API request to GetAllSignups with query parameters to view all signups
+                    #region Step13
+                    Console.WriteLine("Step 13: Get All Signups");
+
+                    // Create additional signups
+                    for (int i = 0; i < AdditionalSignupsToCreate; i++)
+                    {
+                        // The signup does not need to be active to view in Get All Signups. 
+                        CreateSignupInput additionalSignupToCreate = new CreateSignupInput()
+                        {
+                            Name = $"{SignupName}_{i}",
+                            ResourceType = ResourceType.Stream,
+                            ResourceIds = streamsToAdd,
+                        };
+
+                        using StringContent additionalSignupToCreateString = new (JsonSerializer.Serialize(additionalSignupToCreate));
+                        additionalSignupToCreateString.Headers.ContentType = new MediaTypeHeaderValue("application/json");
+                        response = await httpClient.PostAsync(new Uri($"{resource}/api/{apiVersion}/Tenants/{tenantId}/Namespaces/{namespaceId}/signups", UriKind.Absolute), additionalSignupToCreateString).ConfigureAwait(false);
+                        CheckIfResponseWasSuccessful(response);
+                    }
+
+                    response = await httpClient.GetAsync(new Uri($"{resource}/api/{apiVersion}/Tenants/{tenantId}/Namespaces/{namespaceId}/signups?skip={GetAllSignupsSkip}&count={GetAllSignupsCount}", UriKind.Absolute)).ConfigureAwait(false);
+                    CheckIfResponseWasSuccessful(response);
+
+                    SignupsWrapper signupsWrapper = JsonSerializer.Deserialize<SignupsWrapper>(await response.Content.ReadAsStreamAsync().ConfigureAwait(false), _apiJsonOptions);
+
+                    if (signupsWrapper!.Signups.Any())
```

Comment:
It looks like you are checking if there are any members of the ```Signups``` enumerable, but the foreach loop would just be skipped if there are no members so there is no need to check.

Reply:
Resolved.